### PR TITLE
[RAT-4] Create confing package (#1)

### PR DESCRIPTION
**Pull request by Alencion**
* [RAT-4] Add SwaggerConfig in config package

* [RAT-4] Add Swagger2 dependency in pom.xml

### CODE REVIEW

**Review by rinha7 — APPROVED** [review]
GJ

File: src/main/java/org/cnu/realcoding/riotgamesapi/config/SwaggerConfig.java [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+        return Predicates.or(
+                regex("/.*"),
+                regex("/health"),
+                regex("/info")
+        );
+    }
+
+    private ApiInfo apiInfo() {
+        return new ApiInfoBuilder()
+                .title("Weather Crawler")
```
Comment (DoDaek) [inline src/main/java/org/cnu/realcoding/riotgamesapi/config/SwaggerConfig.java]: Weather Crawler?? please change the title

**Review by DoDaek — CHANGES_REQUESTED** [review]
Weather Crawler?? Please change the title

**Review by DoDaek — COMMENTED** [review]
LGTM

**Review by DoDaek — APPROVED** [review]

**Pull request merged**